pkg/local_workflows/network_utils/snyk_request_id.go

```diff
@@ -0,0 +1,16 @@
+package network_utils
+
+import (
+	"github.com/google/uuid"
+
+	"github.com/snyk/go-application-framework/pkg/networking"
+)
+
+func AddSnykRequestId(n networking.NetworkAccess) {
```

Comment:
A godoc comment would be nice 🙏. There's a recommended convention for these, see https://go.dev/doc/comment

(At some point, we should turn on the linter rule requiring them for exported symbols.)